test/JDBC/src/test/java/com/sqlsamples/TestQueryFile.java

```diff
@@ -345,11 +345,11 @@ private static boolean compareOutFiles (File outputFile, File expectedFile) {
         String expectedFilePath = expectedFile.getAbsolutePath();
         ProcessBuilder diffProcessBuilder;
 
-        // if expected file is generated from SQL Server, do not compare error code and message
```

Comment:
The comment is confusing since it's : 
```
if expected file is generated from SQL Server
```

But actually what we wanted to do is : do not make the flaky time diff affect the test result, so pls distinguish those logic in different places.

Reply:
Updated the comments